tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -103,134 +84,91 @@ const updateSettings = async (targetsConfig, user, contactSummary) => {
   const permissions = settings.permissions;
   permissions.can_aggregate_targets = user.roles;
   await utils.updateSettings({ tasks, permissions, contact_summary: contactSummary }, true);
-  await utils.refreshToGetNewSettings();
+  await commonElements.closeReloadModal();
+  await commonElements.goToBase();
 };
 
 const clickOnTargetAggregateListItem = async (contactId) => {
-  await element(by.css(`.aggregate-detail li[data-record-id="${contactId}"] a`)).click();
-  await helper.waitUntilReadyNative(element(by.id('contacts-list')));
+  await analytics.targetAggregateListItem(contactId).click();
   // wait until contact-summary is loaded
-  await helper.waitUntilReadyNative(element(by.css('.content-pane .meta > div > .card .action-header h3')));
+  await contactsPageObject.contactCard().waitForDisplayed();
 };
 
 describe('Target aggregates', () => {
 
   describe('as a db admin', () => {
+    before(async () => await loginPage.cookieLogin());
+    after(async () => {
+      await browser.deleteCookies();
+      await browser.refresh();
+    });
 
-    afterEach(() => utils.revertDb());
+    afterEach(async () => await utils.revertDb([], true));
```

Comment:
Please add the `forms` exception as a parameter. If we don't add that exception all the config forms will be deleted and the tests that are executed after this one might fail.

``` .js
afterEach(async () => await utils.revertDb([/^form:/], true);
```